lib/auth/tls_test.go

```diff
@@ -588,7 +588,7 @@ func TestManualRotation(t *testing.T) {
 	// this is not going to be a problem in real teleport
 	// as it reloads the full server after reload
 	_, err = testSrv.CloneClient(proxy).GetNodes(ctx, apidefaults.Namespace)
-	require.ErrorContains(t, err, "bad certificate")
+	require.ErrorContains(t, err, "unknown certificate authority")
```

Comment:
```suggestion
	require.ErrorContains(t, err, "certificate")
```

We don't care beyond that, or do we? Same for others.